docs/technical-overview.md

```diff
@@ -0,0 +1,267 @@
+*[➡️ Documentation index](./index.md)*
+
+# Technical Overview
+
+**Table of contents**
+
+* [Fixtures](#fixtures)
+* [Manufacturers](#manufacturers)
+* [Fixture model](#fixture-model)
+* [Fixture features](#fixture-features)
+* [Plugins](#plugins)
+  * [Exporting](#exporting)
+  * [Importing](#importing)
+  * [Export Tests](#export-tests)
+* [Testing](#testing)
+* [UI / Website](#ui--website)
+
+## Fixtures
```

Comment:
I'd maybe put Fixtures and Manufacturers into `index.md`. Or add another topic *Folder structure*?